ExeterTeam25/group-software-project
# Daily riddles

Daily riddle implementation

New json file containing the riddles
Auto generates riddles every day
Can complete riddles when close to the location answer
Cant enter url to get points again for same riddle

To test:

Login as user and make sure a daily riddle shows on homepage
Complete the riddle by spoofing location to near answer (ask me for answer) and make sure it disappears from homepage
Make sure points increases
Go to url of the riddle (e.g. viewRiddle/5) and make sure it shows a message saying riddle already done

Now go into admin and delete the riddle
Go back to homepage and make sure new riddle is generated (note only 2 riddles available atm so may generate same one again).
